README.md

```diff
@@ -1,2 +1,177 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"> </a>
+
+- Mobile Version:
+<img src="./img/gif/mobile.gif">
+
+
+-See mobile version preview:
+<img src="./img/gif/desktop.gif">
+
+-See The Proyect deployment <a href="https://willowy-cupcake-fc3c8b.netlify.app/">Here</a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.<br>
+
+### Prerequisites
+
+In order to run this project you need:<br>
+
+-Have VSCode or another text editor installed. <a href="https://code.visualstudio.com/download">Click to download VSC text editor</a> <br> 
+-Install node package. <a href="https://nodejs.org/en/download/">Click to download node package</a> <br> 
+-Have git installed. <a href="https://git-scm.com/downloads">Click to download git</a> <br> 
+-Create a GitHub repository. <a href="https://github.com/">Click to visit Github Website</a> <br> 
+
+
+### Setup
+
+Clone this repository to your desired folder:<br>
+
+cd my-folder
+  git clone https://github.com/DamianDib9/MV-Capstone-project.git
+
+
+### Install
+
+N/A
+
+### Usage
+<br>
+To use with Visual Studio Code, Sublime text, Atom or any text editor.<br>
+
+
+### Run tests
+<br>
+To run tests, run the following command: <br>
+First of all use git bash over microsoft powershell in order to avoid errors.<br>
+<br>
+Example commands:<br>
+You can use HTML file to use this command line in the console: npx hint .<br>
+You can use CSS file to use this command line in the console: npx stylelint "**/*.{css,scss}"<br>
+
+
+### Deployment
+
+You can deploy this project using:<br>
+gitbash, github, github desktop.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="author"></a>
+
+
+👤 **Damian Dib Bustros**
+
+- GitHub: [@githubhandle](https://github.com/damiandib9)
+- Twitter: [@twitterhandle](https://twitter.com/devbustros)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/damian-dib-bustros-953577261)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[New user interface]**
+- [ ] **[Picture Slideshow]**
+- [ ] **[User forum]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Credits to Cindy Shin for sharing it's template!
+
+Contributions, issues, and feature requests are welcome.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project please share it with your friends.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank all of my family for helping me trough this Microverse course.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [  ] For this section, kindly give appropriate references to real creative designers. Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07).

- The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, kindly do it in the section highlighted 👍

Reply:
I already had that reference to Cindy Shin here in collaborator section...
![image](https://user-images.githubusercontent.com/121461459/227755298-a7ca020f-df37-4c43-9142-76c00b9d9320.png)

But no problem, I'll change it to shis section

Reply:
Hello @DamianDib9 , 

I saw it prior to the review, but you need to add it under the `Acknowledgments` section with the appropriate links so users can also navigate to the original creators.

## Instructions:
![image](https://user-images.githubusercontent.com/50990925/227755758-b8f3340e-4e44-4295-952d-e7aafef89611.png)